R/utils.R

```diff
@@ -53,3 +53,18 @@
 .safe_deparse <- function (expr, ...) {
   paste0(deparse(expr, width.cutoff = 500L, ...), collapse = "")
 }
+
+################################################################################
+# checks
+
+#' @importFrom SummarizedExperiment assays
+.check_abund_values <- function(abund_values, x,
```

Comment:
Just reimport it like this:

```
.check_abund_values <- MicrobiomeExperiment:::.check_abund_values 
```

Then the function from `MicrobiomeExperiment` becomes a single point of thruth